scripts/create_mock_data.py

```diff
@@ -0,0 +1,10 @@
+from factories.expense import ExpenseFactory
+from factories.house import HouseFactory
+
+
+def run():
+    for i in range(500):
+        HouseFactory().save()
+
+    for i in range(500):
```

Comment:
I think we can try and create more than one expense for each House, something like 5 for each house from different months.

Reply:
this creates a random house that has the expense i think its not that relevant how many expenses each house has, correct me if Im wrong

Reply:
I think that in the house(mine) view we will want to show more than one expense for each house.
Maybe even a few for each month will be better

Reply:
changed it take a look https://github.com/redhat-beyond/Xpense/blob/a0663d56bc51cd220a044b56b32d7b396109522c/factories/expense.py#L16

Reply:
Great!!
Thank you